sdlf-utils/pipeline-examples/datalake-workload-management/deploy.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+pflag=false
+rflag=false
+sflag=false
+
+DIRNAME=$(pwd)
+
+usage () { echo "
+    -h -- Opens up this help message
+    -p -- Name of the AWS profile to use for the Account
+    -r -- AWS Region to deploy to (e.g. eu-west-1)
+    -s -- S3 bucket to store artifact
+"; }
+options=':p:r:s:h'
+while getopts $options option
+do
+    case "$option" in
+        p  ) pflag=true; PROFILE=${OPTARG};;
```

Comment:
-s and -t for profile to align with the existing deploy scripts.

Reply:
The sdlf-wlm-integration doesnt require any deployment script changes or additions based on new changes. Standalone solution is kept as is